Formula/bettercap.rb

```diff
@@ -19,14 +19,9 @@ class Bettercap < Formula
   uses_from_macos "libpcap"
 
   def install
-    ENV["GOPATH"] = buildpath
-    (buildpath/"src/github.com/bettercap/bettercap").install buildpath.children
-
-    cd "src/github.com/bettercap/bettercap" do
-      system "make", "build"
-      bin.install "bettercap"
-      prefix.install_metafiles
-    end
+    system "make", "build"
+    bin.install "bettercap"
+    prefix.install_metafiles
```

Comment:
Not necessary.

https://github.com/Homebrew/brew/blob/a95dd7d48588099fae744cf8970c9e7ea25368bd/Library/Homebrew/build.rb#L159